src/stories/StyledDataTable.stories.tsx

```diff
@@ -52,7 +52,12 @@ export const PaymentData: ComponentStory<typeof StyledDataTable> = (args) => {
             }}
           />
         </StyledDataTableRow>
-        <StyledDataTableRow label="Purpose of Payment">
+        <StyledDataTableRow
+          label="Purpose of Payment"
+          infoText={
+            'The purpose of payment remains identical for the selected asset and can be used for recurring payments and standing orders.'
```

Comment:
This wording is technically not correct, the purpose does change based on selected asset & iban. And if you change your currency, it might change the destination bank too. Don't know on the detail level we want to describe this